Jenkinsfile

```diff
@@ -13,7 +13,7 @@ node('rhel8'){
 	}
 
 	stage('Build') {
-		env.JAVA_HOME="${tool 'openjdk-1.8'}"
+		env.JAVA_HOME="${tool 'openjdk-1.11'}"

```

Comment:
value took from https://gitlab.mw.lab.eng.bos.redhat.com/search?group_id=&project_id=2178&repository_ref=master&search=openjdk&search_code=true#L762

Reply:
should it be `openjdk-11` only?